### PR TITLE
chore: bump superset-ui and update references to queriesData

### PR DESCRIPTION
**Pull request by villebro**
💔 Breaking Changes
Bump to latest version of `superset-ui-core` to pull in breaking changes introduced in https://github.com/apache-superset/superset-ui/pull/869 and https://github.com/apache-superset/superset-ui/pull/874 . Changes:
- Reference `queriesData` instead of deprecated `queryData`.
- Add new legacy Time sections.
- Add missing deps required by latest version of `chart-controls`.

### CODE REVIEW

File: packages/superset-ui-legacy-preset-chart-kepler/src/KeplerChartPlugin.js [hunk 1]
```diff
@@ -16,8 +16,8 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import { t } from '@superset-ui/translation';
-import { ChartMetadata, ChartPlugin } from '@superset-ui/chart';
+import { t } from '@superset-ui/core';
+import { ChartMetadata, ChartPlugin } from '@superset-ui/core';
```
Comment (simchaNielsen) [inline packages/superset-ui-legacy-preset-chart-kepler/src/KeplerChartPlugin.js:20]: you can unit these 2 imports

File: packages/superset-ui-plugins-demo/storybook/stories/index.js [hunk 2]
```diff
@@ -2,12 +2,16 @@ import '@babel/polyfill';
 import { setAddon, storiesOf } from '@storybook/react';
 import { withKnobs } from '@storybook/addon-knobs';
 import JSXAddon from 'storybook-addon-jsx';
-import categoricalD3 from '@superset-ui/color/esm/colorSchemes/categorical/d3';
-import sequentialCommon from '@superset-ui/color/esm/colorSchemes/sequential/common';
-import sequentialD3 from '@superset-ui/color/esm/colorSchemes/sequential/d3';
-import { configure } from '@superset-ui/translation';
-import { getCategoricalSchemeRegistry, getSequentialSchemeRegistry } from '@superset-ui/color';
-import { getTimeFormatterRegistry, smartDateFormatter } from '@superset-ui/time-format';
+import categoricalD3 from '@superset-ui/core/esm/color/colorSchemes/categorical/d3';
```
Comment (simchaNielsen) [inline packages/superset-ui-plugins-demo/storybook/stories/index.js:5]: should it be from `/esm/`

File: packages/superset-ui-plugins-demo/storybook/stories/index.js [hunk 3]
```diff
@@ -2,12 +2,16 @@ import '@babel/polyfill';
...
+import sequentialCommon from '@superset-ui/core/esm/color/colorSchemes/sequential/common';
+import sequentialD3 from '@superset-ui/core/esm/color/colorSchemes/sequential/d3';
+import { configure } from '@superset-ui/core';
+import {
+  getCategoricalSchemeRegistry,
+  getSequentialSchemeRegistry,
+  getTimeFormatterRegistry,
+  smartDateFormatter,
+} from '@superset-ui/core';
```
Comment (simchaNielsen) [inline packages/superset-ui-plugins-demo/storybook/stories/index.js:14]: also you can unit with previous import from `core`

File: packages/superset-ui-plugins-demo/storybook/stories/legacy-preset-chart-deckgl/Screengrid/Stories.tsx [hunk 4]
```diff
@@ -13,7 +13,7 @@ export default [
         width={400}
         height={400}
         datasource={dummyDatasource}
-        queryData={payload}
+        queriesData={payload}
```
Comment (simchaNielsen) [inline packages/superset-ui-plugins-demo/storybook/stories/legacy-preset-chart-deckgl/Screengrid/Stories.tsx:16]: should be array

**Review by simchaNielsen — APPROVED** [review]

File: packages/superset-ui-plugins-demo/storybook/stories/legacy-preset-chart-deckgl/Screengrid/Stories.tsx (hunk 4 above)
Comment (villebro) [reply packages/superset-ui-plugins-demo/storybook/stories/legacy-preset-chart-deckgl/Screengrid/Stories.tsx:16]: Thanks, good catch!

File: packages/superset-ui-plugins-demo/package.json [hunk 5]
```diff
@@ -41,6 +41,8 @@
     "@types/storybook__react": "3.0.7",
     "bootstrap": "^4.3.1",
     "react": "^16.6.0",
+    "react-bootstrap": "^0.33.1",
+    "react-loadable": "^5.5.0",
```
Comment (villebro) [inline packages/superset-ui-plugins-demo/package.json:45]: These had apparently been made peer dependencies between 0.15 and 0.16.

**Review by rusackas — APPROVED** [review]
LGTM!

**Pull request merged**